app/views/reports/new/modal/_module_contents_inner.html.erb

```diff
@@ -74,6 +74,11 @@
             <%= form.check_box "module_repository_#{repository.id}", label: repository.name.capitalize, data: { id: repository.id } %>
           </li>
         <% end %>
+        <% RepositorySnapshot.with_deleted_parent_by_team(@project.team).each do |repository| %>
```

Comment:
Please move it to controller or helper

Reply:
And snapshots will be shown separately from live repositories?

Reply:
Yes, is this a problem? And the have extra label "(Deleted)", as Sasa suggested.

Reply:
And I would keep it here. Will be hard to find these helpers methods because we render those views in this dynamic magic controller methods..